extractors/register_parser.py

```
@@ -158,7 +175,43 @@ def parse_ocr_text_for_registration(ocr_text):
+            for page_num in range(min(3, len(doc))):
+                page = doc[page_num]
+                pix = page.get_pixmap(dpi=150)  # 낮은 해상도로 빠르게 확인
+                image = pixmap_to_bgr(pix)
+                
+                try:
+                    response = ocr_google_vision(image)
+                    if response.full_text_annotation:
+                        page_text = response.full_text_annotation.text
+                        if "등기사항" in page_text:
+                            return True
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)